app-text/xpdf/xpdf-3.04.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Open source Portable Document Format (PDF) viewer"
+DESCRIPTION="The Xpdf project also includes a PDF text extractor, PDF-to-PostScript \
+converter, and various other utilities.
+
+Xpdf runs under the X Window System on UNIX and OS/2.  The non-X \
+components (pdftops, pdftotext, etc.) also run on Windows and Mac OSX \
+systems and should run on pretty much any system with a decent C++ \
+compiler.  Xpdf will run on 32-bit and 64-bit machines."
+HOMEPAGE="http://www.foolabs.com/xpdf"
+COPYRIGHT="1996-2014 Glyph & Cog, LLC."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://mirror.physik-pool.tu-berlin.de/tex-archive/support/xpdf/xpdf-$portVersion.tar.gz"
+CHECKSUM_SHA256="11390c74733abcb262aaca4db68710f13ffffd42bfe2a0861a5dfc912b2977e5"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [ $targetArchitecture != x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
```

Comment:
Oops, **my previous comment is incorrect**. It **_would_** have been correct **_if_** (and only if) **`ARCHITECTURES`** had a **`!x86_gcc2`**, but it has a **`x86_gcc2`**.

If the recipe works fine on x86_gcc2 primary arch then you can (and propably should) simply **remove** the
```
if [ $targetArchitecture != x86_gcc2 ]; then
	commandSuffix=
	commandBinDir=$prefix/bin
fi
```